packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/OvaVirtualMachinesList.tsx

```diff
@@ -21,12 +21,12 @@ export const ovaVmFieldsMetadataFactory: ResourceFieldFactory = (t) => [
   },
   {
     resourceFieldId: 'concerns',
-    jsonPath: '$.concerns',
+    jsonPath: (data: VmData) => (data.vm.providerType === 'ova' ? data?.vm?.concerns : []),
```

Comment:
why are you using a method ?
this looks like very easy json path, what am i missing ?

Reply:
As discussed in [previous PR](https://github.com/kubev2v/forklift-console-plugin/pull/742#discussion_r1340506261) the main reason is type safety.
We can switch here to string if you prefer.

Reply:
works for me